src/metabase/lib/binning.cljc

```diff
@@ -69,54 +71,49 @@
     x]
    (available-binning-strategies-method query stage-number x)))
 
-(defn default-auto-bin
+(mu/defn default-auto-bin :- ::lib.schema.binning/binning-option
   "Returns the basic auto-binning strategy.
 
   Public because it's used directly by some drill-thrus."
   []
-  {:display-name (i18n/tru "Auto bin")
+  {:lib/type     :option/binning
+   :display-name (i18n/tru "Auto bin")
```

Comment:
Once of the drill thru impls was calling `default-auto-bin` but it didn't produce a valid binning-option resulting in a broken drill